app/jinja_filters.py

```diff
@@ -102,19 +100,14 @@ def format_start_end_date(start_date, end_date=None):
 
 @blueprint.app_template_filter()
 def format_str_as_date_range(value):
-    from_date = format_str_as_date(value['from'])
-    to_date = format_str_as_date(value['to'])
+    from_date = format_date(value['from'])
+    to_date = format_date(value['to'])
     return '{from_date} to {to_date}'.format(from_date=from_date, to_date=to_date)
 
 
 @blueprint.app_template_filter()
 def format_str_as_month_year_date(value):
-    return datetime.strptime(value, '%m/%Y').strftime('%B %Y')
-
-
-@blueprint.app_template_filter()
-def format_str_as_date(value):
-    return format_date(datetime.strptime(value, '%d/%m/%Y'))
+    return format_date(value, '%B %Y')
```

Comment:
- Now we have a consistent date format I think `str` should be removed from method names, so this should be `format_month_year_date`. This would also then be consistent with `format_date` and `format_date_range`.
- Does it matter that the date will now be output in a `<span>` with a `date` class? (this applies to any methods that have been changed to use `format_date`)

Reply:
the `<span>` stops the date line breaking in the middle, which i think it something we want to keep across the board